examples/gno.land/r/demo/memeland/memeland.gno

```diff
@@ -0,0 +1,40 @@
+package memeland
+
+import (
+	"gno.land/p/demo/memeland"
+	"strconv"
+	"time"
+)
+
+var m = memeland.NewMemeland()
+
+func PostMeme(data, timestamp string) string {
+	if m == nil {
+		return "memeland does not exist"
+	}
```

Comment:
When is this situation even possible?

Reply:
I don't think these checks are needed.

Reply:
Ok you guys beat me. I removed them: [efb53ca](https://github.com/gnolang/gno/pull/1751/commits/efb53ca5488a9e9fa20167cb96c1062f1b7ca78c)